src/encoded/static/components/file.js

```diff
@@ -11,6 +11,7 @@ import * as globals from './globals';
 import { requestFiles, requestObjects, requestSearch, RestrictedDownloadButton, AlternateAccession } from './objectutils';
 import { ProjectBadge } from './image';
 import { QualityMetricsPanel } from './quality_metric';
+import { PickerActions } from './search';
```

Comment:
This is David's code, these overwrites must be everywhere.

Reply:
I left out a PickerActions call from File search, so he added it in. Its absence didn’t affect the usual search, but it broke file searches within forms.